client/src/handlers/handlers.ts

```diff
@@ -448,7 +488,11 @@ export function cmdTests(
     ) {
         return async function handler () {
             const flixFilename = await getFlixFilename(context, launchOptions)
-            const cmd = ['java', '-jar', flixFilename, 'test']
+            const jvm:string = vscode.workspace.getConfiguration('flix').get('extraArgumentsToTheJVM')
+            const cmd = ['java']
+            if(jvm.length != 0)
+                cmd.push(...jvm.split(' '))
```

Comment:
There is a lot of repetition with the above four lines of code. Maybe a helper function would be useful? 

You can also do that in the next PR.

Reply:
a helper function is added. 
Resolved